### DISCUSSION TITLE
Sync Between orbit and iTwin Viewer

### DISCUSSION THREAD

**Discussion by tusharsable98** [iTwin/itwinjs-core#5315] [Q&A]
We are trying to sync the camera View of Orbit 3dm viewer with iTwin viewer We found [getViewVector() ](https://sdk.3dmapping.cloud/22.10.0/reference_docs/classes/Viewer3DM.html#getViewVector)function which gives us the geographic coordinates of viewPosition with respect to 4326 : WGS84, a CRS ,along with view position we also get forward vector, up vector and FOV with [getFieldOfViews2D](https://sdk.3dmapping.cloud/22.10.0/reference_docs/classes/Viewer3DM.html#getFieldOfViews2D).We want to use these coordinates in the iTwin viewer to set up the camera position but [lookAt](https://www.itwinjs.org/v2/reference/imodeljs-frontend/views/viewstate3d/lookat/?term=lookat)() in iTwin needs Project coordinates(Spatial Coordinate system) in meters I guess to place the camera.We wanted some direction to move forward on this part.

**Comment by pmconne** [comment]
[Cartographic](https://www.itwinjs.org/reference/core-common/geometry/cartographic/) represents a point in WGS84 coordinates.
[IModelConnection.cartographicToSpatial](https://www.itwinjs.org/reference/core-frontend/imodelconnection/imodelconnection/cartographictospatial/) can convert that into the iModel's spatial coordinate system.
